docs/book/src/tasks/installation.md

```diff
@@ -50,18 +50,36 @@ kubectl cluster-info
 Depending on the infrastructure provider you are planning to use, some additional prerequisites should be satisfied
 before getting started with Cluster API.
 
+<aside class="note">
+
+<h1>Note</h1>
+
+Throughout this quickstart, we've given instructions on setting parameters using environment variables.
+For most environment variables in the rest of the guide, you can also set them in ~/.cluster-api/clusterctl.yaml
+
+</aside>
+
 {{#tabs name:"tab-installation-infrastructure" tabs:"AWS,Azure,Docker,GCP,vSphere,OpenStack,Metal3"}}
 {{#tab AWS}}
 
 Download the latest binary of `clusterawsadm` from the [AWS provider releases] and make sure to place it in your path.
 
+
 ```bash
+$ export AWS_REGION=us-east-1 # This is used to help encode your environment variables
 # Create the base64 encoded credentials using clusterawsadm.
 # This command uses your environment variables and encodes
 # them in a value to be stored in a Kubernetes Secret.
-export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm alpha bootstrap encode-aws-credentials)
+$ export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm alpha bootstrap encode-aws-credentials)
+# This is the name of the SSH key in the EC2 console in
+# the region you are going to place a cluster
+$ export AWS_SSH_KEY_NAME=default
+# Select instance types
+$ export CONTROL_PLANE_MACHINE_TYPE=t3.large
+$ export NODE_MACHINE_TYPE=t3.large
```

Comment:
Would it make sense to set the variables used by the cluster templates closer to their use so it's clear which ones are used for installing the provider vs creating the workload cluster from template?

Reply:
Yes, gimme a sec

Reply:
Ended up rewriting a chunk to make it flow effectively. Note I've change the parameters as per https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/1631 and https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/436

CC @CecileRobertMichon